common-schema.json

```diff
@@ -37,6 +37,12 @@
         },
         "dataProvider": {
           "type": "string"
+        },
+        "owner": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "format": "uri"
```

Comment:
it is not necessary for it to be a URI, it could be a URI or an id (reference) pointing to another entity stored in the same server

Reply:
i added support for both using oneOf construct. alternatively i can simply remove the formatting specification.